client/src/components/Settings/Settings.module.scss

```diff
@@ -26,6 +26,7 @@
     line-height: 1.4rem;
 
     @media #{$tablet-up} {
+      font-family: 'Inter Tight', sans-serif;
```

Comment:
I suggest creating a mixin instead. `fontBig` or something like that, that combines `font-size` & `font-family`. That way we know given size requires given family.

Reply:
Done